File: Python/Easy/MaxConsecutiveOnes.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        c = h = 0
        for x in nums:
            if (x == 1):
                c+=1
            else:
                if (c > h):
                    h = c
                c = 0
        return max(h,c)

File: Java/Easy/MaxConsecutiveOnes.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int c = 0, h = 0;

        for (int x : nums){
            if (x == 1){
                c++;
            } else {
                if (c > h){
                    h = c;
                }
                c = 0;
            }
        }
        return Math.max(h,c);
    }
}
